projects/packages/forms/src/contact-form/class-contact-form-plugin.php

```diff
@@ -572,6 +572,10 @@ protected static function get_outlined_style_attributes( $block_name, $attrs ) {
 			$attributes_for_outlined_style['style']['border'] = $attrs['style']['border'];
 		}
 
+		if ( isset( $attrs['borderColor'] ) ) {
```

Comment:
Oh poor communication on my part - I'd already done that in https://github.com/Automattic/jetpack/pull/43531/files#diff-8553b10e9f86366e46982a8e498ca06c0784d5e64c168a19ed759c2be4b2a986R582

Since this is exactly the same, plus you've improved the notched classes spacing, let's get this one in!